sqlmesh/core/audit/definition.py

```diff
@@ -272,6 +273,7 @@ class StandaloneAudit(_Node, AuditMixin):
     depends_on_: t.Optional[t.Set[str]] = Field(default=None, alias="depends_on")
     hash_raw_query: bool = False
     python_env_: t.Optional[t.Dict[str, Executable]] = Field(default=None, alias="python_env")
+    default_catalog: t.Optional[str] = None
```

Comment:
should this just go in the base node?

Reply:
Nah nodes don't necessarily have a concept of catalog. They can represent things that exist entirely outside of a database.